keps/sig-apps/2232-suspend-jobs/README.md

```diff
@@ -491,29 +499,33 @@ details). For now, we leave it here.
 _This section must be completed when targeting beta graduation to a release._
 
 * **How does this feature react if the API server and/or etcd is unavailable?**
-
-* **What are other known failure modes?**
-  For each of them, fill in the following information by copying the below template:
-  - [Failure mode brief description]
-    - Detection: How can it be detected via metrics? Stated another way:
-      how can an operator troubleshoot without logging into a master or worker node?
-    - Mitigations: What can be done to stop the bleeding, especially for already
-      running user workloads?
-    - Diagnostics: What are the useful log messages and their required logging
-      levels that could help debug the issue?
-      Not required until feature graduated to beta.
-    - Testing: Are there any tests for failure mode? If not, describe why.
-
-* **What steps should be taken if SLOs are not being met to determine the problem?**
-
--->
+  kube-controller-manager will also be unavailable if the API server or etcd
```

Comment:
I would remove this sentence, saying it will be "unavailable" is not accurate, the binary itself might still be running and healthy, but it won't receive job updates and it can't do all the other stuff mentioned below.

Reply:
>the binary itself might still be running and healthy

Perhaps unavailable is the wrong word, but is it really healthy? Nearly every operation that talks to the API server will fail (including ones that GET the Job). With this sentence, I just want to convey that KCM will not be operational like it normally is.

Reply:
> With this sentence, I just want to convey that KCM will not be operational like it normally is.

This is explained in the rest of the paragraph. Note that the controller does have a dedicated health endpoint, so I want to avoid the confusion between that and the case where the dependent components are not responsive. it is a nit though.

Reply:
ack, I don't removing it - done.